packages/amplication-code-gen-types/src/plugin-events.ts

```diff
@@ -59,4 +50,24 @@ export type Events = {
       modules: CreateServerDotEnvParams["after"]
     ) => Module[];
   };
+  [EventNames.CreateEntityService]?: {
+    before?: (
+      dsgContext: DsgContext,
+      eventParams: CreateEntityServiceParams["before"]
+    ) => CreateEntityServiceParams["before"];
+    after?: (
+      dsgContext: DsgContext,
+      modules: CreateEntityServiceParams["after"]
+    ) => Module[];
+  };
+  [EventNames.CreateEntityBaseService]?: {
```

Comment:
rename to CreateEntityServiceBase

Reply:
@yuval-hazaz  also update the params object : from  CreateEntityBaseServiceParams to: CreateEntityServiceBaseParams